_posts/2020-12-17-solidity-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,240 @@
+If the base lies in one of these ranges and the exponent is small enough, it
+should result in a cheap ``exp`` opcode.
+
+#### ``unchecked`` Blocks
+
+Since checked arithmetic uses more gas, as discussed in the previous section, and
+because there are times when you really want the wrapping behaviour, for example when
+implementing cryptographic routines, we provide a way to disable the checked arithmetic and
+re-enable the previous "wrapping" or "modulo" arithmetic:
+
+Any block of the form ``unchecked { ... }`` uses wrapping arithmetic. You can use
+this special block as a regular statement inside another block:
+
+```solidity
+contract C {
+    function f() public pure returns (uint) {
+        uint x = 0;
+        unchecked { x--; }
+        return x;
+    }
+}
+```
+
+An unchecked block can of course contain any number or any kind of statement
+and will create its own variable scope.
+If there is a function call inside the unchecked block, the function will
+not inherit the setting - you have to use another ``unchecked`` block inside the function
+if you also want it to use wrapping arithmetic.
+
+We decided for some restrictions for the ``unchecked`` block:
+
+1. In modifiers you cannot use ``_;`` inside it, because this might lead to confusion about whether the property is inherited.
+
+2. You can only use it inside a block but not as a replacement for a block. The following code snippets, for example, are both invalid:
+
+    ```solidity
+    function f() unchecked { uint x = 7; }
+    ```
+
+    ```solidity
+    function f() pure {
+        uint x;
+        for (uint i = 0; i < 100; i++) unchecked { x += i; }
+    ```
+
+You have to wrap the unchecked blocks in another block to make it work.
+
+
+### Explicit Conversions
+
+Explicit conversions are only possible when there
+is at most one change in sign, width or type-category (``int``, ``address``, ``bytesNN``, etc.).
+
+For example, converting ``int8`` to ``uint16`` is only possible, if you go through ``uint8`` or ``int16``.
+Since the choice of the intermediate conversion will have an effect on the result
+we wanted to make it explicit:
+
+- ``uint16(int16(int8(-1)))`` results in ``0xffff``, while
+- ``uint16(uint8(int8(-1)))`` is ``0xff``.
+
+There are more such changes in relation to payable addresses, literals, enums and contracts. For the full list
+of changes, please see the [documentation](https://docs.soliditylang.org/en/latest/080-breaking-changes.html).
+
+### General Remarks
+
+Since we usually do not backport bugfixes, it is recommended to upgrade all code to be compatible with Solidity v0.8.0.
+
+The [solidity-upgrade tool](https://docs.soliditylang.org/en/latest/using-the-compiler.html#solidity-upgrade) can help you to semi-automatically upgrade your contracts to breaking language changes. While ``solidity-upgrade`` carries out a large part of the work, your contracts will most likely need further manual adjustments.
```

Comment:
`solidity-upgrade` does not do any upgrades for `080`. Maybe we can remove this paragraph?